packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -1,7 +1,7 @@
 import {Entity, model, property} from '@loopback/repository';
```

Comment:
I think `Entity` needs to be changed to `<%= modelBaseClass %>`.

Reply:
Will these be valid?
`import {Entity, property} from '@loopback/repository';`
and
`import {Model, property} from '@loopback/repository';`

Wen use the `<%= modelBaseClass %>` ?